src/vs/workbench/services/userDataProfile/browser/userDataProfileManagement.ts

```diff
@@ -121,7 +121,7 @@ export class UserDataProfileManagementService extends Disposable implements IUse
 			case WorkbenchState.WORKSPACE:
 				return { configPath: workspace.configuration!, id: workspace.id };
 		}
-		return 'empty-window';
+		return { id: workspace.id };
```

Comment:
Why not use `toWorkspaceIdentifier` method here which seems to do the same?

Reply:
It seems `toWorkspaceIdentifier` returns `undefined` when workspace is passed for empty workspace.

Reply:
Yeah true, its a weird one.